platform/mac/do_mac_bundle.sh

```diff
@@ -135,23 +154,46 @@ install_name_tool -change ${BREW}/webp/lib/libwebp.7.dylib libs/libwebp.7.dylib
 
 # prepare bundle for distribution
 ln -s /usr/bin/tar tar
-mv COPYING ../COPYING.txt
+mv COPYING README.md ../Resources/
+mv koreader ../MacOS/koreader
 rm -rf cache clipboard history ota \
     l10n/.git l10n/.tx l10n/templates l10n/LICENSE l10n/Makefile l10n/README.md \
     plugins/SSH.koplugin plugins/hello.koplugin plugins/timesync.koplugin \
     plugins/autofrontlight.koplugin resources/fonts resources/icons/src \
     resources/kobo-touch-probe.png resources/koreader.icns rocks/bin \
-    rocks/lib/luarocks screenshots spec tools README.md
+    rocks/lib/luarocks screenshots spec tools
+
+# adjust reader.lua a bit.
+
+sed '1d' reader.lua > tempfile
+sed -i.backup 's/.\/reader.lua/koreader/' tempfile
+sed -i.backup 's/the last viewed document will be opened"/" .. os.getenv("HOME") .. " will be opened"/' tempfile
+mv tempfile reader.lua
+rm -f tempfile*
+chmod -x reader.lua
 popd || exit 1
 
+# bundle translations, if any
+for i in $(ls l10n); do
+    if [ "${i}" != "templates" ]; then
+        strings_file="l10n/${i}/mac.strings"
+        if [ -f "${strings_file}" ]; then
+            echo "bundling translation for $i"
+            mkdir -p "${APP_PATH}/Contents/Resources/${i}.lproj"
+            cp -p "${strings_file}" "${APP_PATH}/Contents/Resources/${i}.lproj/MainMenu.strings"
+        fi
+    fi
+done
+
 # package as DMG if create-dmg is available
 # reduces size from 80MB to 40MB
 mv "${APP_PATH}" "${APP_BUNDLE}.app"
-if command_exists "create-dmg"; then
-    # create KOReader-$VERSION.dmg with KOReader.app inside
-    create-dmg "${APP_BUNDLE}.app" --overwrite
-    rm -rf "${APP_BUNDLE}.app"
-else
-    # rename as KOReader-$VERSION.app
-    mv -v "${APP_BUNDLE}.app" "${APP_BUNDLE}-${VERSION}.app"
-fi
+
+#if command_exists "create-dmg"; then
```

Comment:
What's the problem with this?

Reply:
That was me commenting the create-dmg stuff to check changes faster! Will fix soon.